az-hadoop-jobtype-plugin/src/main/java/azkaban/jobtype/HadoopJobUtils.java

```diff
@@ -15,6 +15,8 @@
  */
 package azkaban.jobtype;
 
+import azkaban.spi.Storage;
```

Comment:
They aren't. Good catch :smile:

On a separate PR we can run intellij's optimize imports and it can clean up a gazillion unused imports across the code base. I didn't want to do that for this PR though because it would have made the diff enormous.

Reply:
Can you create a followup JIRA to track this?